module/purpose/PS3Walk/src/PS3Walk.cpp

```diff
@@ -134,69 +139,86 @@ namespace module::purpose {
                                 head_locked = !head_locked;
                             }
                             break;
-                            // dance moves here:
-                        case BUTTON_DPAD_UP:
+                        // case BUTTON_DPAD_UP:
```

Comment:
It'd be nice to have this set up with a yaml list of button/script pairs, then we won't have this large block of commented out code.

Reply:
Added, I've also disabled by default because I found myself accidentally triggering scripts when walking robot around